src/main/resources/zodiac.config.yml

```diff
@@ -1,12 +1,101 @@
-$extends: zodiac_base
 subsystems:
```

Comment:
WARNING: regression on this file. Check this out to master version before merging.